plugin/pkg/admission/serviceplan/changevalidator/admission.go

```diff
@@ -121,7 +121,12 @@ func (d *denyPlanChangeIfNotUpdatable) SetInternalServiceCatalogInformerFactory(
 	d.scLister = scInformer.Lister()
 	spInformer := f.Servicecatalog().InternalVersion().ServicePlans()
 	d.spLister = spInformer.Lister()
-	d.SetReadyFunc(scInformer.Informer().HasSynced)
+
+	readyFunc := func() bool {
+		return scInformer.Informer().HasSynced() && scInformer.Informer().HasSynced() && spInformer.Informer().HasSynced()
```

Comment:
We can remove the redundant second call to scInformer.Informer().HasSynced(). That looks to have been a mistype.

Reply:
yeah, that's a mistype